### DISCUSSION TITLE
When using Pixi to draw PCB graphics, the quantity is 100k, and the shape is circular. Is there a faster method?

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#8354] [Q&A]
When using Pixi to draw PCB graphics, the shape is circular, 
the number of elements can reach more than 100k,
 and the maximum number of elements can reach 300K. 
With such a large number of elements, the rendering speed
 will become slow and  the memory consumption will be large.

Is there a faster way to draw a circle
 (I don't know if I can cache the circle, but the position changes)
![image](https://user-images.githubusercontent.com/89827437/169564329-e3c442ad-4373-4036-8b30-1398b590d7d4.png)

**Comment by bigtimebuddy** [comment]
Circles can be particularly expensive because of the number of points/triangulation required to render. The suggestion is to share Graphics geometry so you only have to triangulate it once. This is much more efficient. Here's an example:

https://jsfiddle.net/bigtimebuddy/o3wv1ktp/
